tests/lib/snaps/test-snapd-service/meta/hooks/configure

```diff
@@ -4,11 +4,7 @@ snapctl set service-option="$(snapctl get service-option-source)"
 
 COMMAND=$(snapctl get command)
 if [ "$COMMAND" != "" ]; then
-    if [ "$COMMAND" = "restart" ]; then
```

Comment:
is this change required? is this fixing/simplifying the test?

Reply:
It's not required, but it seems the `if` and `else` result in entirely identical behavior, so not sure why they're differentiated.

Reply:
In my opinion probably best to avoid actual changes here since this'll probably end up with the "Skip spread" tag, although this does look correct and simpler.